backend/climateconnect_api/utility/common.py

```diff
@@ -1,14 +1,16 @@
-from climateconnect_api.models.role import Role
 from climateconnect_api.models import Skill
+from climateconnect_api.models.role import Role
+
 
 def get_skill_name(skill: Skill, language_code: str) -> str:
-    if language_code in ["de"]:
-        return getattr(skill, "name_{}_translation".format(language_code))
-    if language_code == "en":
+    a = getattr(skill, "name_{}_translation".format(language_code))
+    if language_code == "en" or a == None:
         return skill.name
+    return a
+
 
 def get_role_name(role: Role, language_code: str) -> str:
-    if language_code == "en":
+    a = getattr(role, "name_{}_translation".format(language_code))
+    if language_code == "en" or a == None:
         return role.name
-    else:
-        return getattr(role, "name_{}_translation".format(language_code))
\ No newline at end of file
+    return a
```

Comment:
This change makes a lot of sense! However I think for unrelated changes it could make sense to quickly create an additional PR which we can then instantly merge. This way small fixes aren't blocked by larger features

Reply:
Nevermind. This doesn't work for english because there is no name_en_translation attribute. Therefore we'll need to go back to the old code or find a different solution